libs/sdk-ui-tests-e2e/package.json

```diff
@@ -32,44 +32,41 @@
         "styles"
     ],
     "config": {
-        "eslint": "-c .eslintrc.js --ext ts,tsx src/"
+        "eslint": "-c .eslintrc.js --ext ts,tsx cypress/"
     },
     "scripts": {
-        "clean": "rm -rf ci dist esm coverage *.log styles/css && jest --clearCache",
+        "clean": "rm -rf ci dist esm coverage *.log styles/css",
         "api-extractor": "mkdir -p api && [ -z \"${CI}\" ] && (api-extractor run -l) || (api-extractor run)",
-        "build": "bash scripts/build.sh && npm run api-extractor",
-        "build-dev": "bash scripts/build.sh --dev",
-        "build-cjs": "tsc -p tsconfig.build.json",
-        "build-esm": "tsc -p tsconfig.build.esm.json",
+        "build": "tsc -p tsconfig.build.json",
         "dev": "bash scripts/build.sh --dev-watch",
-        "test": "jest --watch",
-        "test-once": "jest",
-        "test-ci": "JEST_JUNIT_OUTPUT=./ci/results/test-results.xml jest --ci --config jest.ci.js",
         "eslint": "eslint $npm_package_config_eslint",
         "eslint-ci": "mkdir -p ./ci/results && eslint -f checkstyle -o ci/results/eslint-results.xml $npm_package_config_eslint",
         "prettier-check": "prettier --check '{src,stories,styles,__mocks__}/**/*.{ts,tsx,json,scss,md,yaml,html}'",
         "prettier-write": "prettier --write '{src,stories,styles,__mocks__}/**/*.{ts,tsx,json,scss,md,yaml,html}'",
-        "dep-cruiser": "depcruise --validate .dependency-cruiser.js --output-type err-long src/",
-        "dep-cruiser-ci": "depcruise --validate .dependency-cruiser.js --output-type err-long src/",
+        "dep-cruiser": "depcruise --validate .dependency-cruiser.js --output-type err-long cypress/",
+        "dep-cruiser-ci": "depcruise --validate .dependency-cruiser.js --output-type err-long cypress/",
         "validate": "npm run dep-cruiser && npm run eslint && npm run prettier-check",
-        "validate-ci": "npm run dep-cruiser-ci && npm run eslint-ci && npm run prettier-check"
+        "validate-ci": "npm run dep-cruiser-ci && npm run eslint-ci && npm run prettier-check",
+        "cypress": "node ./scripts/run_cypress_local.js"
     },
     "dependencies": {
         "lodash": "^4.17.19",
         "react-intl": "^3.6.0",
-        "tslib": "^2.0.0"
+        "tslib": "^2.0.0",
+        "dotenv": "^8.2.0"
```

Comment:
Please keep the dependencies sorted (rush does not do this for some reason). Same for devDependencies.